content/en/dashboards/scheduled_reports.md

```diff
@@ -73,7 +73,7 @@ From the configuration modal that opens, you can pause an existing report or cre
 
 ## Permissions
 
-Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles. 
+Users with the Admin role or `Org Management` permission can enable or disable scheduled reports for an account from the **Settings** tab under [Public Sharing][14] in **Organization Settings**. Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles.
```

Comment:
> can enable or disable scheduled reports for an account 

can we update this ^ wording to the following? (you can pause individual reports, so this clarifies that the setting turns off the whole feature):

> can enable or disable the scheduled reports feature for an account